<ui version="4.0" >
 <class>msg_journal_settings_form</class>
 <widget class="QWidget" name="msg_journal_settings_form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Настройки функционирования журнала распоряжений</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <widget class="QGroupBox" name="gbFilters" >
     <property name="title" >
      <string>Фильтры</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QGroupBox" name="gbInputMsg" >
        <property name="title" >
         <string>Входящие сообщения</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <widget class="QCheckBox" name="cbInputMsg" >
           <property name="text" >
            <string>Показывать</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbInputEx" >
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <item>
             <widget class="QCheckBox" name="cbInputMsgDate" >
              <property name="text" >
               <string>Поступившие в указанную дату:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="deInputMsg" />
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbOutputMsg" >
        <property name="title" >
         <string>Исходящие сообщения</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <widget class="QCheckBox" name="cbOutputMsg" >
           <property name="text" >
            <string>Показывать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbOutputEx" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout" >
            <item>
             <widget class="QCheckBox" name="cbOutputMsgDate" >
              <property name="text" >
               <string>Выданные в указанную дату</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="deOutputMsg" />
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSettings" >
     <property name="title" >
      <string>Настройки</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Обновлять журнал каждые</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbInterval" >
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>минут</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
